.drone.yml

```diff
@@ -19,6 +19,7 @@ pipeline:
     pull: true
     commands:
       - cd /drone/fed-server
+      - git clone https://github.com/owncloud/testing
```

Comment:
wrong target - right?

Reply:
yup - but as we use owncloud-core plugin - this should no longer be required as the core plugin already fetches owncloud.